### ISSUE TITLE
Code documentation

### ISSUE THREAD

**Issue by hartbit** [weissi/swift-server-logging-api-proposal#51]
The public API should probably be more thoroughly documented.

**Comment by weissi** [comment]
Definitely, will tackle that as soon as we deleted everything that’s not actually part of the package. Lots of the code is just examples.
